easybuild/easyconfigs/l/libpsl/libpsl-0.21.1-GCCcore-11.2.0.eb

```
@@ -0,0 +1,32 @@
+easyblock = 'ConfigureMake'
+
+name = 'libpsl'
+version = '0.21.1'
+
+homepage = 'https://rockdaboot.github.io/libpsl'
+description = "C library for the Public Suffix List"
+
+toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
+
+source_urls = ['https://github.com/rockdaboot/libpsl/archive']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['ac6ce1e1fbd4d0254c4ddb9d37f1fa99dec83619c1253328155206b896210d4c']
+
+builddependencies = [
+    ('binutils', '2.37'),
+    ('libtool', '2.4.6'),
+    ('Autotools', '20210726'),
+    ('CMake', '3.22.1'),
```

So.. this stuck out as odd to me. There is no cmake here at all. And, why libtool and Autotools? It's not generating any configure script..  I tried to look it up and i can't see any reason why any of these were added. Removing all 3 made no difference in my test. I see i bumped this last time and probably just overlooked it, and before that  it was added without any explanation.

lets get rid of it. I do however that Python was dropped as a dep.. but this still looks like it ought to be a builddep. 

```suggestion
    ('Python', '3.9.6'),
```

---

Sounds good to me. As that was an update from an existing EC file, I just took it as it is. I have added `Python` but I added it as the `-bare` one, as it is only required as a build-dependency. Maybe that is the reason why that was not discovered before.